### ISSUE TITLE
Windows: jadx-gui-0.8.0.exe can't be placed in the Open With context menu

### ISSUE THREAD

**Issue by jpstotz** [skylot/jadx#377]
On Windows 10 I tried to add the `jadx-gui-0.8.0.exe` in the Open With list of APK files.

However it does not matter what I do Jad-X never appears in this context menu even if it you do open an APK using `jadx-gui-0.8.0.exe` once. Other apps then appear in the open with context menu, but not JadX.
Really strange. There must be something different/wrong with the `jadx-gui-0.8.0.exe` that prevents Windows from displaying it in the open with list.

**Comment by jpstotz** [comment]
I searched the net for this problem and found many similar issues in other applications. 

It seems to be a problem with Windows 10 and the included app meta data in the EXE file. WIndows seems to assume that every EXE file has at least `CompanyName` and a `FileDescription` entry that is not empty.

For jad-x this means that the launch4j configuration options should include a `companyName = "....."`  entry. It seems like the FileDescription is automatically added by launch4j. In my personal build it worked after adding the comanyName line. 

I would provide a pull request, but I have to admit that I don't have a clue what we should set as companyName for an open source software like jad-x.

In my tests I used `companyName = "Open Source Software (Apache License 2.0)"`

Any comments or other suggestions?
